.gitignore

```diff
@@ -3,3 +3,4 @@ _artifacts/
 lima.REJECTED.yaml
 schema-limayaml.json
 .config
+.idea/
```

Comment:
Please don't put your personal config into the project-specific settings files.

This configuration belongs into your `~/.gitignore` file, so it applies to all your repos.